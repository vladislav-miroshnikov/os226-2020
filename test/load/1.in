echo hello from side-loaded echo
retcode
