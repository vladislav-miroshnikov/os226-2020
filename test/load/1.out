hello from side-loaded echo
4
